unitTests/ui/MissingTermsTest.ts

```diff
@@ -244,6 +280,41 @@ export function MissingTermsTest() {
             expect(getMissingTerms()).toEqual([KoreanWordPresentMultipleTimes]);
           });
         });
+        describe('when a words contains words boundary character', () => {
+          it(`when the missing term is part of the words,
+          it will not be displayed as a missing term.`, () => {
+            const wordBoundarysCharacter = [
+              `-`,
+              `'`,
+              `?`,
+              `*`,
+              `’`,
+              `.`,
+              `~`,
+              `=`,
+              `,`,
+              `/`,
+              `\\`,
+              `:`,
+              `\``,
+              `;`,
+              `_`,
+              `!`,
+              `&`,
+              `(`,
+              `)`
+            ];
+            let query: string;
```

Comment:
Same comments as above apply to this test (out-of-scope variable and variable name).